Sources/Subs-Auth.php

```diff
@@ -855,11 +856,23 @@ function smf_setcookie($name, $value = '', $expire = 0, $path = '', $domain = ''
 		$httponly = !empty($modSettings['httponlyCookies']);
 	if ($secure === null)
 		$secure = !empty($modSettings['secureCookies']);
+	if ($samesite === null)
+		$samesite = !empty($modSettings['samesiteCookies']) ? $modSettings['samesiteCookies'] : 'lax';
```

Comment:
The documentation states that SameSite=None must be "Secure".  Can you add a logic check for that?

Reply:
@albertlast You marked this resolved.  I'm not sure I'm reading how?

Reply:
FYI,  The documentation I'm reading from is here:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite#samesitenone_requires_secure

Should be simple to implant in the logic to enforce if samesite=none to enforce secure.